sympy/physics/tests/test_units.py

```diff
@@ -20,6 +20,8 @@ def test_units():
     t = Symbol('t')
     assert integrate(t*m/s, (t, 1*s, 5*s)) == 12*m*s
     assert (t * m/s).integrate((t, 1*s, 5*s)) == 12*m*s
+    
+    assert (year / week).evalf(15) == Float(52.17745571428571)
```

Comment:
Am 27.03.2015 um 23:36 schrieb Jason K. Moore:
> Secondly, all this PR then needs is quotes around the float value:
> `Float('52.17745571428571')` and there is no issue.

The issue actually stays. This just moves the floating-point conversion 
from the Python compiler to the Float constructor.

If a test is supposed to be in, I'd suggest testing against an exact value.
Looking at the definition of ```year``` (that's a few lines below 
```week```), it is defined as ```tropical_year``` which is 
```Rational('365.24219')*day```. That would be exact because 
```Rational('365.24219')``` isn't going through a floating-point conversion.
Using ```common_year``` would be even easier, that's just ```365``` and 
the test would not depend on any conversion considerations.

Or we could just compare against, say, ```day * 7```.
That would be silly, but it would test exactly what the definition of 
```week``` is supposed to be.